### ISSUE TITLE
Add Wallabag instance to app using a qr code

### ISSUE THREAD

**Issue by jrabensc** [wallabag/ios-app#346]
| Q             | A
| ------------- | ---
| Device?       | iPhone
| Device Os version    | 14.6
| App Version   | 6.0.4 build 304
| Wallabag server version | 2.4.2

### Steps to reproduce/test case

Add new Wallabag instance to the app.

### Improvement

Inserting the two strings (client id, secret) is prone to error and hard to debug if something fails. It would be nice if we could use the qr code provided within Wallabag or to manually create one to be read by the iOS app.

**Comment by bourvill** [comment]
ref #244

**Comment by bourvill** [comment]
For the moment it is difficult to have a fix on the server side.

I just had an idea. Maybe we could use the VisionFramework to read the string. In the same way that credit cards are read by applications

**Comment by jrabensc** [comment]
Yes that would be nice. The login process altogether is a bit rough, but I understand this is due to limitations in the server.
